test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -45,7 +45,19 @@ def test_ec2_tensorflow_inference_eia_gpu(tensorflow_inference_eia, ec2_connecti
     run_ec2_tensorflow_inference(tensorflow_inference_eia, ec2_connection, "8500", region)
 
 
-def run_ec2_tensorflow_inference(image_uri, ec2_connection, grpc_port, region):
+@pytest.mark.model("mnist")
+@pytest.mark.parametrize("ec2_instance_type", ["p2.xlarge"], indirect=True)
+def test_ec2_tensorflow_inference_gpu_telemetry(tensorflow_inference, ec2_connection, region, gpu_only):
+    run_ec2_tensorflow_inference(tensorflow_inference, ec2_connection, "8500", region, True)
+
+
+@pytest.mark.model("mnist")
+@pytest.mark.parametrize("ec2_instance_type", ["c5.4xlarge"], indirect=True)
+def test_ec2_tensorflow_inference_cpu_telemetry(tensorflow_inference, ec2_connection, region, cpu_only):
+    run_ec2_tensorflow_inference(tensorflow_inference, ec2_connection, "8500", region, True)
+
+
+def run_ec2_tensorflow_inference(image_uri, ec2_connection, grpc_port, region, Telemetry_mode=False):
```

Comment:
nit: Could it be better to set the first character of it not to capital, a.k.a "telemetry_mode"?